elektronLogging/loggerChain.go

```diff
@@ -0,0 +1,28 @@
+package elektronLogging
+
+import (
+	log "github.com/sirupsen/logrus"
+	"os"
+)
+
+type Logger interface {
+	SetNext(logType Logger)
+	Log(logType int, level log.Level, logData log.Fields, message string)
```

Comment:
As fields are optional, it is better to either have them as the last argument or not have them in the argument list at all.
I have created [this sample code](https://play.golang.org/p/RILTxcZtT2w) for reference.

Reply:
Done